peribolos/knative.yaml

```diff
@@ -877,8 +877,8 @@ orgs:
           Knative Release Leads:
             description: Active member of the Knative Release Leads rotation.
             maintainers:
-            - yanweiguo
-            - lionelvillard
+            - ZhiminXiang
+            - evankanderson
```

Comment:
I think the next issue has now been done, so I believe we can remove that entry now and solely rely on the Knative Release Leads and no special docs permissions are required.

Reply:
not sure I understand. Is there anything to do other than removing Richie (which I did)?

Reply:
Nope, that was all I thought we could now safely do. I just jotted it first down in the first file, then pointed to place where (which you did in this commit) the change needed to be made.
Thanks and sorry for the confusing verbiage :)